mock/py/mockbuild/backend.py

```diff
@@ -742,7 +742,9 @@ def get_command(mode):
                     self.buildroot.root_log.info("Dynamic buildrequires detected")
                     self.buildroot.root_log.info("Going to install missing buildrequires")
                     buildreqs = glob.glob(bd_out + '/SRPMS/*.buildreqs.nosrc.rpm')
-                    self.installSrpmDeps(*buildreqs)
+                    result = self.installSrpmDeps(*buildreqs)
+                    if "Nothing to do" in result:
```

Comment:
Please run "rpm -qa | wc -l" before and after in chroot and compare....

Reply:
No idea how.

Reply:
`self.buildroot.doChroot()`

Reply:
on it